scalajvm/app/views/main.scala.html

```diff
@@ -24,6 +24,6 @@
             <li class="menu-service"><a href="@routes.SuggestedQuotes.newQuote()">Добавить цитату</a></li>
 	    </ul>
         <div class="content center-content">@content</div>
-        <div class="footer center-content"><span class="highlight"><a href="https://github.com/codingteam/loglist">codingteam</a> ©</span> 2015</div>
+        <div class="footer center-content"><span class="highlight"><a href="https://github.com/codingteam/loglist">codingteam</a> ©</span> 2016</div>
```

Comment:
How bout ranges?

Reply:
I find it funny that the guy who said "This is why my copyright notice for this site […] does not include a year" now started including a copyright year on the named site.